app/views/static_pages/board.html.erb

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html >
+<html>
+<style type="text/css">
+ 	.board {
+ 		width: 640px;
+    	height: 655px;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
+ 	}
+ 	body {
+ 		 padding-top: 30px; 
+ 		 padding-bottom: 30px;
+ 		
+ 	}
+	td {
+ 		empty-cells: show;
+ 		height: 80px;
+ 		width: 80px;
+ 		vertical-align:middle;
+ 	}
+ 	table.one {
+ 		border-collapse: collapse;
+ 	}
+ 	td.black {
+ 		background-color: #000608;
+ 	}
+ 	</style>
+	<body>
+	<div class="board">
+ 		<table class="one">
+ 			<!-- 1st Row -->
+ 			<tr>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 	</tr>
+			<!-- 2nd Row -->
+ 			<tr>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 			</tr>
+			<!-- 3rd Row -->
+				<tr>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 	</tr>
+ 			<!-- 4th Row -->
+ 			<tr>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 			</tr>
+ 			<!-- 5th Row -->
+ 				<tr>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 	</tr>
+ 		 	<!-- 6th Row -->
+ 		 	<tr>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 			</tr>
+ 			<!-- 7th Row -->
+ 				<tr>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 	</tr>
+ 		 	<!-- 8th Row -->
+ 		 	 	<tr>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 		 		<td></td>
+ 		 		<td class="black"></td>
+ 			</tr>
+ 			</table>
+ 		</div>
+	</body>
+</html>
```

Comment:
Just add a carriage return after `</html>`. Basically just need a blank line at the end of the page ;)